### PR TITLE
Add Xilinx

### LINKED ISSUE #3422: Request: Verilog/Xilinx Vivado
This is a bit of a weird request, since there's not a lot of concrete symbols out there for this. However, this icons list has no VHDL icons available.

**Name:** Verilog (or Xilinx Vivado)
**Website:** There's not a concrete Verilog site, though there IS the popular IDE [Xilinx Vivado](xilinx.com) 
**Alexa rank:** 38,837 for the above site.

So, there's two possibilities that can be included here. The symbol for Xilinx Vivado's IDE:
https://www.logolynx.com/images/logolynx/96/96d4637c642f165b67a52408f99c2255.png
This has a main color of #EE3626 and an off color of #B41E3C.

Or, the VSCode Icons supported SystemVerilog icon (which is already in an SVG):
https://github.com/vscode-icons/vscode-icons/issues/1442
This is just purely black and white, so the main color would be #000000.

I'm wondering whether to implement both in a pull request, or if one of the icons is unnecessary/against contribution guidelines. I'm planning to submit a pull request with the implementation for both, however.

EDIT: To be clear, if anyone wants to implement this, feel free - Otherwise, I'll do a pull request myself.

### PR DESCRIPTION
**Pull request by sharmavins23**
This is a part of my original request for HDL oriented icons. This adds the branded Xilinx icon, creators of Xilinx Vivado (one of the more popular SystemVerilog and Verilog IDEs.)

xilinx

**Issue:** Closes #3422 
**Alexa rank:** 38,837

  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

The color was picked based on the two major colors that the logo originally had. This was the main color (not the shadow color.)
I vectorized the logo with Inkscape.

### CODE REVIEW

**Comment by NovaGL** [comment]
Thanks for this. Where did you get the logo from or did you convert it from a PNG yourself.

Why did you choose that color? I would have gone with `#ec1c24` myself

**Comment by sharmavins23** [comment]
I went ahead and converted from a PNG. The color came from a screen select of that PNG's main color as well.

**Comment by NovaGL** [comment]
@sharmavins23 

I managed to get a vector version of this logo and the color choices available are 
`#ed1c29` or `#b30838`

Here is the path I used
`M0.3,6l5.2-6h7.7L8,6 M0.3,6l5.2,6h7.7L8,6 M8,18l5.2-6H5.6l-5.2,6 M0.3,18l5.2,6h7.7L8,18 M13.3,0l5.2,6 l5.2-6 M23.7,24l-5.2-6l-5.2,6`

It can be found in this file
https://www.xilinx.com/support/documentation/user_guides/ug116.pdf

**Comment by sharmavins23** [comment]
@NovaGL added your changes, awaiting CI and approval

File: icons/xilinx.svg
Comment (adamrusted) [inline icons/xilinx.svg:1]: Hi @sharmavins23, thanks for your work on this!
I'm still seeing some slight differences between your version (red) and source (black). They have an SVG in the header of their website, which is what I've used for my source - may be worth rebuilding to that?
image

I've included my SVG below, which passes the linter, should you want to use that one:
```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Xilinx icon</title><path d="M8 18l5.241 6H5.586L.345 18l5.241-6L.345 6l5.241-6h7.655L8 6l5.241 6L8 18zM23.655 0H13.241l5.241 6 5.173-6zM13.241 24h10.414l-5.172-6-5.242 6z"/></svg>
```

I can also see a statement regarding their buyout by AMD - @simple-icons/maintainers should we still look to include this, on the assumption that AMD keep the brand going?

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by PeterShaggyNoble** [comment]
Welcome to Simple Icons, @sharmavins23 and thanks for the contribution 🙂

Further to the comments already made by the rest of the team above, I think we should also consider `#171c2d` for the brand colour as that was my first instinct on visiting their website.

@adamrusted, I think, in the absence of any statement to the contrary from either company, that we should work under the assumption that Xilink will continue to exist as a separate brand.

**Comment by adamrusted** [comment]
@PeterShaggyNoble looking through the publications [here](https://www.xilinx.com/news/media-kits.html) and the favicon of the website, I'd suggest the red is more synonymous with the brand, especially when all their hardware is bright red too. Happy for another @simple-icons/maintainers to say otherwise though 😅

**Comment by adamrusted** [comment]
xilinx

@simple-icons/maintainers due to lack of response by @sharmavins23, I've merged the newer path in that I'd suggested above, and pulled the hex value from the SVG on the homepage directly.

**Comment by fbernhart** [comment]
@adamrusted Perfect! @sharmavins23 thanks as well for the initial work on this icon! 🏆

**Pull request merged**
